dashboard/app/models/script_level.rb

```diff
@@ -2,16 +2,17 @@
 #
 # Table name: script_levels
 #
-#  id         :integer          not null, primary key
-#  script_id  :integer          not null
-#  chapter    :integer
-#  created_at :datetime
-#  updated_at :datetime
-#  stage_id   :integer
-#  position   :integer
-#  assessment :boolean
-#  level_id   :integer
-#  properties :text(65535)
+#  id          :integer          not null, primary key
+#  level_id    :integer
```

Comment:
It was re-ordered in `schema.rb` during @balderdash's PR here: https://github.com/code-dot-org/code-dot-org/pull/8295/files#diff-11195b8114efb5813026789c8fbfbe64L559.  My guess is that `ScriptLevel` hasn't been re-annotated until now.